kubeflow/components/filter-paraphrase.yaml

```diff
@@ -0,0 +1,136 @@
+name: Filter Paraphrases
+description: |
+  Filter paraphrases using a parser
+inputs:
+  - {name: image, description: '', default: ''}
+  - {name: s3_bucket, description: '', default: ''}
+  - {name: owner, description: '', default: ''}
+  - {name: task_name, description: '', default: '' }
+  - {name: project, description: '', default: ''}
+  - {name: experiment, description: '', default: ''}
+  - {name: dataset, description: '', default: ''}
+  - {name: s3_input_datadir, description: '', default: ''}
+  - {name: filtering_model, description: '', default: ''}
+  - {name: paraphrasing_model, description: '', default: ''}
```

Comment:
You don't need the paraphrasing model at this step, correct?

Reply:
Right, fixed.